server/graphql/v1/CollectiveInterface.js

```diff
@@ -1188,6 +1188,22 @@ const CollectiveFields = () => {
       description: 'Returns whether this host is trusted or not',
       resolve: collective => Boolean(get(collective, 'data.isTrustedHost')),
     },
+    isTwoFactorAuthEnabled: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      description: 'Returns whether this user has two factor authentication enabled',
+      async resolve(collective) {
+        if (collective.type === types.USER) {
+          const user = await models.User.findOne({
+            attributes: ['id', 'twoFactorAuthToken'],
+            where: { CollectiveId: collective.id },
+          });
+          if (user.twoFactorAuthToken) {
+            return true;
+          }
+        }
+        return false;
+      },
+    },
```

Comment:
Let's not expose whether users have 2FA enabled to the entire world. Needs to be root or admin of the profile.

Reply:
Ah, missed this. 👍🏽 Corrected.